doc/ref/helpintf.xml

```diff
@@ -109,6 +103,8 @@ no further regulations for the format of the <F>manual.six</F> file,  other  tha
 what you yourself impose. If such a line is missing then it is assumed  that
 the <F>manual.six</F> file complies with the <F>gapmacro.tex</F> documentation  format
 which is the <C>default</C> format.
```

Comment:
I still don't like the <C>default</C> format - is is possible to rephrase that as I've already suggested?